cmd/plume/upload.go

```diff
@@ -50,8 +51,8 @@ var (
 func init() {
 	cmdUpload.Flags.StringVar(&bucket, "bucket", "gs://coreos-plume", "gs://bucket/prefix/")
```

Comment:
If we make the default `gs://users.developer.core-os.net/$USER/$BOARD/$VERSION` instead then the translation to a GCE name doesn't have to handle the special case for default paths vs user-specified paths.